spec/helpers.lua

```diff
@@ -974,12 +1002,12 @@ return {
     local running_conf = conf_loader(default_conf.kong_env)
     if not running_conf then return end
 
-    -- kill kong_tests.conf services
-    for _, pid_path in ipairs {running_conf.nginx_pid,
-                               running_conf.serf_pid} do
-      if pl_path.exists(pid_path) then
-        kill.kill(pid_path, "-TERM")
-      end
+    -- kill kong_tests.conf service
+    local pid_path = running_conf.nginx_pid
+    if pl_path.exists(pid_path) then
+      kill.kill(pid_path, "-TERM")
+      wait_pid(pid_path, timeout, true)
     end
+
```

Comment:
Unrelated change? Curious what the motivation is.

Reply:
which unrelated change, removing `serf_pid`?

Reply:
new line, whitespace